JSTests/test262/expectations.yaml

```diff
@@ -1102,9 +1102,9 @@ test/intl402/DateTimeFormat/timezone-not-canonicalized.js:
 test/intl402/DurationFormat/prototype/format/precision-exact-mathematical-values.js:
   default: 'Test262Error: Duration is {"seconds":10000000,"nanoseconds":1} Expected SameValue(«0:00:10,000,000.000000002», «0:00:10,000,000.000000001») to be true'
   strict mode: 'Test262Error: Duration is {"seconds":10000000,"nanoseconds":1} Expected SameValue(«0:00:10,000,000.000000002», «0:00:10,000,000.000000001») to be true'
-test/intl402/DurationFormat/prototype/format/rounding-mode-trunc-for-seconds.js:
-  default: 'Test262Error: Intl.DurationFormat should format seconds, milliseconds and microseconds with `roundingMode: "trunc"` Expected SameValue(«2», «1») to be true'
-  strict mode: 'Test262Error: Intl.DurationFormat should format seconds, milliseconds and microseconds with `roundingMode: "trunc"` Expected SameValue(«2», «1») to be true'
+test/intl402/DurationFormat/prototype/formatToParts/formatToParts-style-digital-en.js:
```

Comment:
This change has introduced a new FAIL, which I believe is due to rounding errors from adding seconds as a double. In other words, I don't think this change is the direct cause. For example, the failure in the `precision-exact-mathematical-values.js` test is for the same reason.

To fundamentally solve the issue, we need to add seconds as an integer rather than as a double. For this, I am waiting for https://github.com/tc39/test262/pull/3988.

Reply:
Sounds good.